### ISSUE TITLE
Add an option to specify the exact date of expiration

### ISSUE THREAD

**Issue by Otiel** [sebsauvage/ZeroBin#46]
It would be nice if we had an option to choose the **exact date of expiration**.

For instance, there could be a new item for the "Expires" option called "Specify" that would open a calendar component. The user would choose a date, and data would be saved to expire on 23:59 of this date (or 00:00 ?).

**Comment by ckegelman** [comment]
I agree also it would be nice to know where to change the options for the dropdown
